cdds/cdds/prepare/command_line.py

```diff
@@ -205,11 +207,26 @@ def parse_generate_args(arguments: List[str]) -> Namespace:
         'request', help=(
             'The full path to the configuration file containing the information about the request.'
         ))
-
+    parser.add_argument(
+        'plugin_id', help=(
+            'The ID of the plugin for that the mappings configurations are.'
+        ))
     parser.add_argument(
         '-r', '--reconfigure', action='store_true', help=(
             'Replace the MIP Convert config files generated by the configure step')
     )
+    parser.add_argument(
+        '--plugin_module_path', help=(
+            'If you use a external plugin: Path to the module that contains the implementation of the plugin '
+            'that should be loaded'
+        )
+    )
+    parser.add_argument(
+        '--plugin_location', help=(
+            'If you use a external plugin: Path to the plugin implementation that should be added to the '
+            'PYTHONPATH'
+        )
+    )
```

Comment:
This information should be in the request file so I'd prefer them not to be allowed as command line arguments